packages/shopify-codemod/transforms/function-to-arrow.js

```diff
@@ -1,14 +1,19 @@
 export default function functionToArrow({source}, {jscodeshift: j}, {printOptions = {}}) {
   return j(source)
     .find(j.FunctionExpression)
-    .filter((path) => j(path).find(j.ThisExpression).size() === 0 && !j.Property.check(path.parent.node))
+    .filter(({parent}) => !j.MethodDefinition.check(parent.node) && !j.Property.check(parent.node))
+    .filter((path) => j(path).find(j.ThisExpression).size() === 0)
```

Comment:
Can we combine these into a single `filter`? You can still have separate functions for each check and just `&&` them together in the filter:

```js
function isConvertableFunction(path) {
  return !isMember(path) && !containsThisExpression(path)
}

.filter(isConvertibleFunction)
```